.github/workflows/build.yml

```diff
@@ -4,6 +4,9 @@ name: build
 on:
   workflow_dispatch:
   pull_request:
+    # ready_for_review is not included in the default types for pull_request, and since we're skipping
+    # building in draft mode, we want that so that when a draft pr is marked ready, we run everything
```

Comment:
This comment might be a bit confusing since `ready_for_review` is in the types list below. Perhaps rephrasing it might help.